### PR TITLE
Tasks2.3 & 2.4 done.

### PR DESCRIPTION
**Pull request by oksana-gorovaya**

### CODE REVIEW

File: average.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+const first_number = 1;
+const second_number = 7;
+const range = _.range(first_number, second_number +1);
+let amount = 0;
+let numbers_sum = 0;
+
+range.forEach(function(element){
+    if (element % 3 === 0){
+        amount += 1;
```
Comment (ulfheddin) [inline average.js]: (minor) maybe better 'count' or 'quantity'? 'amount' sounds a bit off

File: average.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+арифметическое всех чисел из отрезка [a;b], которые делятся на 3.
+
+В приведенном ниже примере среднее арифметическое считается для чисел на отрезке [−5;12]. Всего чисел, делящихся на 3,
+    на этом отрезке 6: −3,0,3,6,9,12. Их среднее арифметическое равно 4.5
+
+На вход программе подаются интервалы, внутри которых всегда есть хотя бы одно число, которое делится на 3.﻿*/
+
+$first_number = 1;
+$second_number = 7;
+$range = range($first_number, $second_number +1);
```
Comment (ulfheddin) [inline average.php]: (minor) space formatting around "+"

**Review by ulfheddin — APPROVED** [review]

File: average.js (hunk 1 above)
Comment (oksana-gorovaya) [reply average.js]: Поменяла на counter.

File: average.php (hunk 2 above)
Comment (oksana-gorovaya) [reply average.php]: Applied.

**Pull request merged**